integration.py

```diff
@@ -0,0 +1,219 @@
+import subprocess
+import unittest
+
+
+class TestPanbyteIntegration(unittest.TestCase):
```

Comment:
**question: ** Why did you choose python to write integration tests? I think it was doable in java as well.